common/lib/xmodule/xmodule/course_module.py

```diff
@@ -506,9 +506,9 @@ class CourseFields(object):
     )
     cert_name_long = String(
         help=_(
-            "Use this setting only when generating PDF certificates. "
-            "Between quotation marks, enter the long name of the course to use on the certificate that students "
-            "receive when they complete the course."
+            'Use this setting only when generating PDF certificates. '
+            'Between quotation marks, enter the long name of the type of certificate that students '
+            'receive when they complete the course. For instance, "Verified Certificate".'
```

Comment:
The system sometimes suffixes `cert_name_long` with "verified," so someone following the suggestion here will cause the system to generate strings like "Verified Verified Certificate." I suggest using the [system default](https://github.com/edx/edx-platform/blob/master/lms/envs/common.py#L2177-L2179), "Certificate of Achievement", as the example `cert_name_long`.